File: src/main/java/ru/vaschenko/ParkPoint/dto/ParkingSpaceDTO.java
package ru.vaschenko.ParkPoint.dto;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Setter @Getter
public class ParkingSpaceDTO {
    private Long id;
    private ParkingZoneDTO parkingZoneDTO; //
    private Integer order;
    private Integer price;
    private Boolean isAvailable;
    private String description;
}

File: src/dto/ParkingSpaceDto.js
export class ParkingSpaceDto {
    constructor(id, idOwner, order, price, isAvailable, parkingZoneDto, bookings) {
      this.id = id;
      this.idOwner = idOwner;
      this.order = order;
      this.price = price;
      this.isAvailable = isAvailable;
      this.parkingZoneDto = parkingZoneDto;
    }
  }
  